uber/site_sections/registration.py

```diff
@@ -500,7 +500,7 @@ def register(self, session, message='', **params):
                 message = 'Please select a payment type'
             elif attendee.payment_method == c.MANUAL and not re.match(c.EMAIL_RE, attendee.email):
                 message = 'Email address is required to pay with a credit card at our registration desk'
-            elif attendee.badge_type not in [c.ATTENDEE_BADGE, c.ONE_DAY_BADGE]:
+            elif attendee.badge_type not in [badge for badge, desc in c.AT_THE_DOOR_BADGE_OPTS]:
```

Comment:
Hm, I think we may want to implement that in the magprime repo, particularly since these badge types don't actually have dates associated with them after you exit the loop in AT_THE_DOOR_BADGE_OPTS. Unless you think it's easy to re-associate them?